Src/library/vision2/interface/events/ev_dpi_action_sequence.e

```diff
@@ -0,0 +1,60 @@
+note
+	description: "Summary description for {EV_DPI_ACTION_SEQUENCE}."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	EV_DPI_ACTION_SEQUENCE
+inherit
+	EV_ACTION_SEQUENCE [TUPLE [dpi:INTEGER; x: INTEGER; y: INTEGER; width: INTEGER; height: INTEGER]]
```

Comment:
in many place, the strict style is not respected, not always critical, but for the main class interface, please add a space after the `:`

Reply:
Done